libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -58,6 +61,11 @@ public class EntitlementInitialization {
 
     private static ElasticsearchEntitlementChecker manager;
 
+    interface InstrumentationInfoFunction {
```

Comment:
merge conflicts ahead? but suppose you are aware ... https://github.com/elastic/elasticsearch/pull/122232/files/8399edb4225a53f6811a0502334ae47c7bd543d0#diff-c6c52e88f05c4d9bcaa9aff17a5119638199235053edd1ef3247119a52521634R71

Reply:
Oh yes, I intentionally duplicated this and intend to resolve the conflict

Reply:
(and take the opportunity to rename)